graylog2-web-interface/src/stores/sidecars/ConfigurationVariableStore.js

```diff
@@ -0,0 +1,114 @@
+import Reflux from 'reflux';
+import URI from 'urijs';
+
+import URLUtils from 'util/URLUtils';
+import UserNotification from 'util/UserNotification';
+import fetch from 'logic/rest/FetchProvider';
+import CombinedProvider from 'injection/CombinedProvider';
+
+const { ConfigurationVariableActions } = CombinedProvider.get('ConfigurationVariable');
+
+const ConfigurationVariableStore = Reflux.createStore({
+  listenables: [ConfigurationVariableActions],
+  sourceUrl: '/sidecar/configuration_variables',
+
+  _fetchConfigurationVariables() {
+    const baseUrl = `${this.sourceUrl}`;
+    const uri = URI(baseUrl).toString();
+
+    return fetch('GET', URLUtils.qualifyUrl(uri));
+  },
+
+  all() {
+    const promise = this._fetchConfigurationVariables();
+    promise
+      .then(
+        () => {},
```

Comment:
If you don't want to use the `then` in the promise, you can call `catch` instead:

`promise.catch((error) => {...})`